### DISCUSSION TITLE
Probleme bei der Erfassung von Daten über Shelly Plug S

### DISCUSSION THREAD

**Discussion by Volkusch** [camueller/SmartApplianceEnabler#486] [Installation und Betrieb]
Hallo,

warum wird die Leistung (kW) und die daraus restituieren Arbeit (kWh) zweier, mit einem Shelly Plug S gemessenen Verbraucher (Espressomaschine u. Waschmaschine), im Sunny Portal gar nicht oder wenn doch, nur ein geringer Anteil, erfasst, in der Shelly App werden alle Werte sofort und korrekt angezeigt?!

Mir ist aufgefallen, dass das, wie in meinem Fall, nur bei den beiden o. g. Verbrauchern der Fall ist, die entweder nur kurzweilig und/oder mit starken kurzweiligen Leistungsschwankungen (10W – 2000W) arbeiten, bei Verbrauchern mit längerfristigen Betriebszeiten mit und längeren Leistungsspitzen-Phasen werden die Wert im Sunny Portal analog zu den Werten in der Shelly App angezeigt.

Probehalber habe ich das einmal mit einem Haarfön und einem Shelly Plug S ausprobiert (kurzweiliger Betrieb und größerer schwankende Leistungsphasen), hier ist mir aufgefallen, dass die aktuellen Werte in der Shelly App sofort korrekt angezeigt werden, im Sunny Portal aber erst mit einer Verzögerung von ca. 105 – 120 Sek. und wenn dann die Leistungsschwankungen oder Betriebszeiten unterhalb des genannten Zeitraumes von 105 – 120 Sek. lagen, wird davon im Sunny Portal überhaupt nichts mehr erfasst!?
Wo liegt der Fehler?

**Comment by camueller** [comment]
> warum wird die Leistung (kW) und die daraus restituieren Arbeit (kWh) zweier, mit einem Shelly Plug S gemessenen Verbraucher (Espressomaschine u. Waschmaschine), im Sunny Portal gar nicht oder wenn doch, nur ein geringer Anteil, erfasst, in der Shelly App werden alle Werte sofort und korrekt angezeigt?!

Siehe:
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp1
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp2

> Mir ist aufgefallen, dass das, wie in meinem Fall, nur bei den beiden o. g. Verbrauchern der Fall ist, die entweder nur kurzweilig und/oder mit starken kurzweiligen Leistungsschwankungen (10W – 2000W) arbeiten, bei Verbrauchern mit längerfristigen Betriebszeiten mit und längeren Leistungsspitzen-Phasen werden die Wert im Sunny Portal analog zu den Werten in der Shelly App angezeigt.

Siehe
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/MeterReportingSwitch_DE.md#z%C3%A4hlerbasierter-zustandsmelder

**Reply by Volkusch** [reply]
Danke für die schnelle Antwort...
Ich glaube, den Fehler gefunden zu haben... Einstellung Schaltertyp "Zählerbasierter Zustandsmelder" und der nicht passende Wert bei der Leistungsgrenze war vermutlich das Problem... Ich werde es die kommenden Tage beobachten.
Danke
